### PR TITLE
chore: Remove lists v1 ff

### LINKED ISSUE #1892: Remove Lists feature flags
## 🎉 Description

This PR removes the lists feature flags and all the feature flagged code.

### PR DESCRIPTION
**Pull request by kevinszuchet**
Closes #1892

### CODE REVIEW

File: webapp/src/components/FavoritesCounter/FavoritesCounter.spec.tsx [hunk 1]
```diff
@@ -146,58 +140,6 @@ describe('FavoritesCounter', () => {
     })
   })
 
-  describe('when the user clicks the component', () => {
-    describe('and the item is unpicked', () => {
```
Comment (LautaroPetaccio) [inline webapp/src/components/FavoritesCounter/FavoritesCounter.spec.tsx:150]: If I'm not wrong, we're removing all the tests that exercise that the onClick is being called. What do you think of keeping two, the last one (testing that the onClick was not called twice) and another one to test that the onClick function was called when clicked.

**Review by LautaroPetaccio — COMMENTED** [review]
The PR looks great! 👍  Just a minor comment on a test

File: webapp/src/components/FavoritesCounter/FavoritesCounter.spec.tsx (hunk 1 above)
Comment (kevinszuchet) [reply webapp/src/components/FavoritesCounter/FavoritesCounter.spec.tsx:150]: Nice catch, thanks!

**Review by LautaroPetaccio — APPROVED** [review]
The best PRs are the ones that remove code 💣

**Pull request merged**